### PR TITLE
Test/workspace api

### PR DESCRIPTION
**Pull request by jsburckhardt**
Validation:

```bash
cd tests/acceptance/azure
export DATABRICKS_HOST="https://****.azuredatabricks.net"
export DATABRICKS_TOKEN="***"
go test -run "TestAzureWorkspace" .   -v
```

image

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,4 +2,11 @@ module github.com/polar-rams/databricks-sdk-golang
 
 go 1.12
 
-require github.com/google/go-querystring v1.0.0
+require (
+	github.com/google/go-querystring v1.0.0
+	github.com/onsi/ginkgo v1.10.2
+	github.com/onsi/gomega v1.5.0
```
Comment (maniSbindra) [inline go.mod:8]: Do we need gomega now that we are using assert

File: tests/acceptance/azure/workspace_test.go [hunk 2]
```diff
@@ -1,25 +1,94 @@
...
+
 func TestAzureWorkspaceList(t *testing.T) {
+	// list and assert
 	const rootPath = "/"
+	listRequest := httpmodels.ListReq{
+		Path: rootPath,
+	}
+	workSpaceList, e := c.Workspace().List(listRequest)
+	assert.Nil(t, e, "could not list workspaces")
+	// fmt.Printf("something %s", &workSpaceList.Objects)
```
Comment (maniSbindra) [inline tests/acceptance/azure/workspace_test.go:26]: can we remove the comment?

File: tests/acceptance/azure/workspace_test.go [hunk 3]
```diff
@@ -1,25 +1,94 @@
...
+	deleteRequest := httpmodels.DeleteReq{
+		Path:      samplePath,
+		Recursive: true,
+	}
+	assert.Nil(t, c.Workspace().Delete(deleteRequest), fmt.Sprintf("could not delete the imported resquest: %s", importRequestPython.Language))
+}
 
-	if len(workSpaceList.Objects) == 0 {
-		t.Error("TestAzureWorkspaceList: Could not retrieve default workspaces")
+func randSeq(n int) string {
```
Comment (maniSbindra) [inline tests/acceptance/azure/workspace_test.go:87]: Should we move this function to new common file for tests like config_test.go or to a new file

File: go.mod (hunk 1 above)
Comment (jsburckhardt) [reply go.mod]: Updated the go.sum file

File: tests/acceptance/azure/workspace_test.go (hunk 2 above)
Comment (jsburckhardt) [reply tests/acceptance/azure/workspace_test.go]: Removed the comment

File: tests/acceptance/azure/workspace_test.go (hunk 3 above)
Comment (jsburckhardt) [reply tests/acceptance/azure/workspace_test.go:86]: @nedruk is looking into that in her branch. I'll include a comment in her tasks to update this code based on her changes

**Comment by maniSbindra** [comment]
lgtm

**Pull request merged**